gobblin-dataset-config/src/main/java/gobblin/config/client/ConfigClient.java

```diff
@@ -0,0 +1,17 @@
+package gobblin.config.client;
+
+import java.net.URI;
+
+import com.typesafe.config.Config;
+
+import gobblin.config.configstore.ConfigStore;
+
+public interface ConfigClient {
```

Comment:
What is this interface? What is the difference to gobblin.dataset.config.ConfigClient?